qiskit/transpiler/passes/routing/lookahead_swap.py

```diff
@@ -85,15 +86,23 @@ def __init__(self, coupling_map, search_depth=4, search_width=4, fake_run=False)
         """LookaheadSwap initializer.
 
         Args:
-            coupling_map (CouplingMap): CouplingMap of the target backend.
+            coupling_map (Union[CouplingMap, Target]): CouplingMap of the target backend.
             search_depth (int): lookahead tree depth when ranking best SWAP options.
             search_width (int): lookahead tree width when ranking best SWAP options.
             fake_run (bool): if true, it only pretend to do routing, i.e., no
                 swap is effectively added.
+            target (Target): A target representing the target backend, if both
+                ``coupling_map`` and this are specified then this argument will take
+                precedence and the other argument will be ignored.
```

Comment:
This extra arg doc wants deleting again.

Reply:
Sigh, I knew I was going to miss one of these. I'm impressed I only missed one docstring in the earlier commit.

Reply:
Fixed in: https://github.com/Qiskit/qiskit-terra/pull/9263/commits/87d0d4078e6846d8ef6824c0797bee567c3423b8